project/profiles/capacity4more/modules/c4m/user/c4m_user_profile/c4m_user_profile.block.inc

```diff
@@ -248,5 +248,9 @@ function c4m_user_profile_get_group_type_title($type) {
     'pending' => t('Membership requested'),
   );
 
+  if (!isset($titles[$type])) {
```

Comment:
It happens while rendering the "my groups" block (`_c4m_user_profile_block_user_my_profile_groups()`), if $account = null.